### ISSUE TITLE
should be able to sync to other human release coordinates

### ISSUE THREAD

**Issue by nathandunn** [GMOD/myvariantviewer#1]
If we can't use the other coordinates, we'll just use the hg19 assembly instead, but this should be configurable in the track.

http://docs.myvariant.info/en/latest/doc/variant_annotation_service.html?highlight=hg38

**Comment by cmdcolin** [comment]
Most databases only have hg19 loaded. Only certain ones offer hg38, clinvar dbnsfp, and evs.

http://docs.myvariant.info/en/latest/doc/data.html

**Comment by monicacecilia** [comment]
@cmdcolin Thanks for noting that. Although that may be the case for now, it will certainly continue to change as the human genome research community continue to produce more data for the latest version of the assembly. GRCh38 was released well over 2 years ago. We must ensure flexibility and adaptability.

**Comment by nathandunn** [comment]
@cmdcolin  The API actually had wisely built-in support for mapping coordinates to hg38, hg19, and hg18 coordinates, so we should also be able to support those alternate assemblies, as well.  I think that by default the names all come from hg19.

**Comment by cmdcolin** [comment]
@nathandunn I think only certain variant annotations contain the position on multiple assemblies. MyGene.info is interesting because you can specifically query hg19 or hg38 coords (mygene.info is hg38 by default) but myvariant.info is hg19.

**Comment by cmdcolin** [comment]
If you want I can just make one that uses the hg38 coords from clinvar

Then you just get the clinvar data in the right place

**Comment by nathandunn** [comment]
Yeah, that might be an option (limiting the variant sources).   However, we have existing hg19 assemblies, so we can just use those if we want to show more than clinvar.   

We have a few options, so when we come back to this, we can evaluate whether we want to see all or just clinvar / dbSNP (what is supported), but we can cross that bridge when we get to it.    

Thanks for clarifying this.
